### DISCUSSION TITLE
[Tests] Add test for the place model

### DISCUSSION THREAD

**Discussion by MxOliver** [Terrastories/terrastories#725] [Temporarily Archived Tickets]
Add tests for the place model (`app/models/place.rb`)

**Comment by vulgofoobar** [comment]
More and less I don't know how to, but I would like to :)

**Comment by noelledusahel** [comment]
@kalimar cool if i pick this up?

**Comment by kalimar** [comment]
@noelledusahel thank you!

**Comment by rgenchev** [comment]
Can I work on this?
